apps/verify/views/content/accessibility.md

```diff
@@ -1,7 +1,7 @@
 # Accessibility statement for Refer family to come to the UK (Afghan Citizens Resettlement Scheme)
 
 
-This accessibility statement applies to the [www.refer-afghan-family-to-come-to-the-uk.homeoffice.gov.uk](www.refer-afghan-family-to-come-to-the-uk.homeoffice.gov.uk) website. It does not cover related pages on the [www.gov.uk](www.gov.uk) website, which has its own accessibility statement.
+This accessibility statement applies to the [www.refer-afghan-family-to-come-to-the-uk,homeoffice.gov.uk](http://www.refer-afghan-family-to-come-to-the-uk.homeoffice.gov.uk) website. It does not cover related pages on the [www.gov.uk](http://www.gov.uk) website, which has its own accessibility statement.
```

Comment:
I think you replaced a `.` with a `,` by accident in the ACRS link text.

also both URLs should be https?